helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -64,11 +64,51 @@ spec:
               hostPort: {{ .Values.chaosDaemon.grpcPort }}
             - name: http
               containerPort: {{ .Values.chaosDaemon.httpPort }}
-      {{- with .Values.chaosDaemon.tolerations }}
-      tolerations:
-{{ toYaml . | indent 8 }}
-      {{- end }}
+{{- if .Values.bpfki.create }}
+        - name: bpfki
+          image: {{ .Values.bpfki.image }}
+          imagePullPolicy: {{ .Values.bpfki.imagePullPolicy | default "IfNotPresent" }}
+          {{- if .Values.bpfki.resources }}
+          resources:
+{{ toYaml .Values.bpfki.resources | indent 12 }}
+          {{- end }}
+          command:
+            - /usr/local/bin/bpfki
+            - -port
+            - !!str {{ .Values.bpfki.grpcPort }}
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: localtime-path
+              mountPath: /etc/localtime
+              readOnly: true
+            - name: modules-path
+              mountPath: /lib/modules
+              readOnly: true
+            - name: src-path
+              mountPath: /usr/src
+              readOnly: true
+            - name: sys-path
+              mountPath: /sys
+              readOnly: true
+          ports:
+            - name: grpc
+              containerPort: {{ .Values.bpfki.grpcPort }}
+              hostPort: {{ .Values.bpfki.grpcPort }}
+{{- end }}
       volumes:
         - name: socket-path
           hostPath:
             path: {{ .Values.chaosDaemon.socketPath | default "/var/run/docker.sock" }}
+        - name: localtime-path
```

Comment:
if we disabled kernel chaos, these volumes are unnecessary. Can we also use `bpfki.create` field to control these volumes?